lib/network/modules/MouseWheelListener.js

```diff
@@ -0,0 +1,34 @@
+/*
+    Auto detects which wheel listener is supported by
+    the current browser, and normalizes the event to
+    the modern 'wheel' api
+*/
+
+let prefix = "",
+_addEventListener,
+support;
+
+if (window.addEventListener) {
+    _addEventListener = 'addEventListener';
+} else {
+    _addEventListener = 'attachEvent';
+    prefix = "on";
+}
```

Comment:
This is IE8 stuff. The oldest browser we support is IE11. Not mentioning the fact that the rest of the codebase doesn't use attachEvent anyway.

This file as whole is useless.